src/status_im/signals/core.cljs

```diff
@@ -75,4 +75,5 @@
       "wallet" (ethereum.subscriptions/new-wallet-event cofx (js->clj event-js :keywordize-keys true))
       "local-notifications" (local-notifications/process cofx (js->clj event-js :keywordize-keys true))
       "community.found" (link.preview/cache-community-preview-data (js->clj event-js :keywordize-keys true))
+      "status.updates.timedout" (visibility-status-updates/handle-visibility-status-updates cofx (js->clj event-js :keywordize-keys true))
```

Comment:
typo ? or intentional ? time**D**out

Reply:
yes, intentional. Status-go is signaling for timed out status updates.
Thank you for reviewing PR.